### PR TITLE
Update update-external-links-gallery.md

### PR DESCRIPTION
**Pull request by jukent**
Added comments to help users fill out the issue template. 
See https://github.com/ProjectPythia/projectpythia.github.io/issues/98

### CODE REVIEW

**Comment by jukent** [comment]
@andersy005 Perhaps we want to leave out the new comments at the end of each line in the generated PR. This is a minor preference. Thoughts?

File: .github/ISSUE_TEMPLATE/update-external-links-gallery.md [hunk 1]
```diff
@@ -5,40 +5,42 @@ title: ''
...
 
-<!-- Please fill out the template below. -->
+<!-- Please fill out the template below by adding the necessary information in between the single quotes in each relevant or known field. -->
 
 Make sure the following items are specified :
 
-- `url`
-- `title`
-- `description`
+- [ ] `url`
```
Comment (clyne) [inline .github/ISSUE_TEMPLATE/update-external-links-gallery.md:12]: How are these checkboxes used? I.e. is the  issue submitter intended to check them, or do they get auto-populated somehow? 

In the case of the former (issue submitter checks them):

1. What happens if they are not checked? Any consequence, or are they just intended to help guide the submitter?
2. My understanding with template check boxes is that they can't be checked until **after** the issue is submitted, which is pretty confusing to the uninitiated :-). We should probably warn them of this.


If the checkboxes serve no purpose other than to remind the user I would suggest not having them given the awkward that GitHub supports them.

File: .github/ISSUE_TEMPLATE/update-external-links-gallery.md [hunk 2]
```diff
@@ -5,40 +5,42 @@ title: ''
...
     affiliation: ''
     affiliation_url: ''
     email: ''
 tags:
   packages:
-    - unspecified
+    - unspecified # List each Python package associated with this resource on a seperate line (the dash is only necessary for the first entry).
   formats:
-    - unspecified
+    - unspecified # List the formats this resource takes (e.g. book, course, documentation, gallery, tutorial, video).
```
Comment (clyne) [inline .github/ISSUE_TEMPLATE/update-external-links-gallery.md:41]: Are their specific keywords that the user needs to choose from for formats, packages or domain? If so, maybe we should be more specific: e.g. you must choose from the following list.

**Review by clyne — APPROVED** [review]
Thanks for cleaning this up, @jukent ! I just have a couple of nits inline

File: .github/ISSUE_TEMPLATE/update-external-links-gallery.md (hunk 1 above)
Comment (jukent) [reply .github/ISSUE_TEMPLATE/update-external-links-gallery.md:12]: The user checks them without any consequence, just works as a guide. I'll change them back.

File: .github/ISSUE_TEMPLATE/update-external-links-gallery.md (hunk 2 above)
Comment (jukent) [reply .github/ISSUE_TEMPLATE/update-external-links-gallery.md:41]: They can submit an entirely new keyword! But these are the keywords currently in play. I will specify this.

File: .github/ISSUE_TEMPLATE/update-external-links-gallery.md [hunk 3]
```diff
@@ -5,40 +5,42 @@ title: ''
...
 
 - `url`
 - `title`
 - `description`
+- `authors` (please credit the creators of the resource in at least one method (`name`, `affiliation`, `affiliation_url`, or `email`)
+- `tag` (at least one tag is required, this helps users access the link by searching via the relevant `packages`, `formats`, or `domains`)
 
 **Note**:
 
 - The thumbnail can be a url to a direct image link.
```
Comment (andersy005) [inline .github/ISSUE_TEMPLATE/update-external-links-gallery.md:20]: With the presence of the comment on line 25, should we remove this?

```suggestion
```

**Comment by andersy005** [comment]
> @andersy005 Perhaps we want to leave out the new comments at the end of each line in the generated PR. This is a minor preference. Thoughts?

I don't think we have to worry about comments. They are stripped out when parsing the issue body. So, we should be good with or without them.

**Comment by jukent** [comment]
Ready to merge?

File: .github/ISSUE_TEMPLATE/update-external-links-gallery.md (hunk 3 above)
Comment (jukent) [reply .github/ISSUE_TEMPLATE/update-external-links-gallery.md]: Yes that is redundant. Thanks.

**Review by andersy005 — APPROVED** [review]

**Pull request merged**